views/layout.hbs

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/public/stylesheets/styles.css">
```

Comment:
In the `app.js` file you already specified that every static elements that you will be using in your website (ex: images, files) will be located inside the `public` folder:

```
app.use(express.static("public"));
```

So when you need to reference the `path` of something that is inside the `public` folder you don't need to specify public in the path you just need to right the path after the public folder:

```
<link rel="stylesheet" href="/stylesheets/styles.css">
```